src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Add comments for this interface. Especially when your interface contains no code, comments are necessary to know what this interface should do.